src/components/VmDetails/cards/UtilizationCard/NetworkingCharts.js

```diff
@@ -31,11 +31,11 @@ import NoLiveData from './NoLiveData'
 const NetworkingCharts = ({ netStats, isRunning, id }) => {
   const haveNetworkStats = !!netStats['current.total']
 
-  const used = (netStats['current.total'] && netStats['current.total'].datum) || 0
+  const used = (netStats['current.total'] && Array.isArray(netStats['current.total'].datum) && netStats['current.total'].datum[0]) || 0
```

Comment:
Any reason to check Array.isArray() here? If so then it should be applied to all places referring to datum[0].